sdks/python/apache_beam/runners/worker/dataflow_element_execution_tracker_test.py

```diff
@@ -0,0 +1,200 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from apache_beam.transforms.cy_combiners import DistributionAccumulator
+from apache_beam.utils.counters import Counter
+from apache_beam.utils.counters import CounterFactory
+from apache_beam.utils.counters import CounterName
+
+
+class DataflowElementExecutionTrackerTest(unittest.TestCase):
+  class _FakeScopedState(object):
+    def __init__(self, sampler, name, state_index, counter=None):
+      self.sampler = sampler
+      self.name = name
+      self.state_index = state_index
+      self.counter = counter
+
+  def setUp(self):
+    self.counter_factory = CounterFactory()
+    try :
+      from apache_beam.runners.worker.dataflow_element_execution_tracker \
+        import DataflowElementExecutionTracker
+      self.tracker = DataflowElementExecutionTracker()
+    except ImportError:
+      self.tracker = None
```

Comment:
<!--new_thread; commit:1f7d05e01de972b0e3bec788fe9c82377ea44f15; resolved:0-->
You can use SkipTest to skip tests that are not in Cython (https://github.com/apache/beam/blob/b095837ff7ee05a9f5b1282aae783ceccce7ec99/sdks/python/apache_beam/runners/worker/statesampler_test.py#L38)

Reply:
Fixed this in the latest commit, thanks!